test/auth/Makefile

```diff
@@ -21,7 +21,7 @@ ORGANIZATION=VMD
 ORGANIZATION_UNIT=VMD
 
 CA_COMMON_NAME=vmd-ca
-VMD_SERVER_COMMON_NAME=vmd-server
+VMD_SERVER_COMMON_NAME=localhost
```

Comment:
what's with this name change to `localhost`? `vmd-server` was better as `localhost` can be confused to name of 127.0.0.1

Reply:
That was because TLS cert requires URL's to match. Is there a better strategy?

Reply:
Does it affect something functionally? If yes, comment would be good. Otherwise, the prev name was more literal on the server common name.

Reply:
Will do!